### PR TITLE
[RemoteEvent][Webhook] Add Mailjet support

### PR DESCRIPTION
**Pull request by blaugueux**
| Q             | A
| ------------- | ---
| Branch       | 6.4
| Bug fix      | no
| New feature  | yes
| Deprecations | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Mailjet/RemoteEvent/MailjetPayloadConverter.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    private function getReason(array $payload): string
+    {
+        if (isset($payload['smtp_reply'])) {
+            return $payload['smtp_reply'];
+        }
+        if (isset($payload['error_related_to'])) {
+            return $payload['error_related_to'];
+        }
+
+        return '';
```
Comment (fabpot) [inline src/Symfony/Component/Mailer/Bridge/Mailjet/RemoteEvent/MailjetPayloadConverter.php]:
```suggestion
        return $payload['smtp_reply'] ?? $payload['error_related_to'] ?? '';
```

**Review by fabpot — APPROVED** [review]
LGTM, just a small suggestion to make code shorter and maybe more readable.

**Review by fabpot — CHANGES_REQUESTED** [review]
@blaugueux You need to add `"symfony/webhook": "^6.3"` in the `require-dev` section of the Mailjet bridge `composer.json` file to make tests pass.

**Comment by fabpot** [comment]
@blaugueux Can you rebase to see if the tests would pass?

File: src/Symfony/Component/Mailer/Bridge/Mailjet/composer.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
         "symfony/mailer": "^5.4.21|^6.2.7|^7.0"
     },
     "require-dev": {
-        "symfony/http-client": "^5.4|^6.0|^7.0"
+        "symfony/http-client": "^5.4|^6.0|^7.0",
+        "symfony/webhook": "^6.3"
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Mailjet/composer.json]:
```suggestion
        "symfony/webhook": "^6.4"
```

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @blaugueux.

**Pull request merged**